### ISSUE TITLE
Generating two files from group even though there is only one js or css is mentioned.

### ISSUE THREAD

**Issue by simpleAdminDeveloper** [wro4j/wro4j#1058]
created two different resource groups that each use wildcards; i.e:
<group name="jsBundle">
  <js>/path/js/**.js</js>
</group>
<group name="cssBundle">
  <css>/path/css/**.css</css>
</group>
these are generating four files as jsBundle.js, jsBundle.css, cssBundle.css, cssBundle.js can anyone help to explaing why it's happening?
I need to generate only one file per group and in each group there will be only one type of file either js or css.

**Comment by alexo** [comment]
This is an expected behavior. Any group can contain any type of resources (both: css & js). 
I assume that jsBundle.css is empty. This is something that could be improved, by not creating any resource if the content is empty...

**Comment by simpleAdminDeveloper** [comment]
Yes jsBundle.css is empty, but cssBundle.js contains ? in second line.
how can I proceed so that these both file will not be generated. Is there any configuration for this.
If not how can I proceed on achieving this.

**Comment by alexo** [comment]
The easiest way to get 2 bundled resources instead of 4, is to create a single group holding both css & js resources. 

It is weird that the cssBundle.js contains a "?" and nothing else. Unless you could provide more details or  a quick start project which I could use to reproduce the problem, there is little I can do.
